ras_backstage/resources/sample/sample.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from flask import jsonify, make_response, request, Response
+from flask_restplus import Resource, reqparse
+from structlog import wrap_logger
+
+from ras_backstage import sample_api
+from ras_backstage.common.filters import get_collection_exercise_by_period
+from ras_backstage.controllers import collection_exercise_controller, sample_controller, survey_controller
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+parser = reqparse.RequestParser()
+parser.add_argument('file', location='files', required=True)
+
+
+@sample_api.route('/<short_name>/<period>')
+class Sample(Resource):
+
+    @staticmethod
+    @sample_api.expect(parser)
+    def post(short_name, period):
+        logger.info('Uploading sample', shortname=short_name, period=period)
+        survey = survey_controller.get_survey_by_shortname(short_name)
+        # TODO: get survey type from service(?)
```

Comment:
Is this todo still valid?

Reply:
No, I don't think it is as we're currently only handling business (B) surveys.